pkg/orchestrator/provider/aws/scanestimation/estimate.go

```diff
@@ -0,0 +1,404 @@
+	scannerVolumeType := params.ScannerVolumeType
+
+	// Get relevant current prices from AWS price list API
+	if sourceRegion != destRegion {
+		// if the scanner is in a different region then the scanned asset, we have another snapshot created in the source region.
+		sourceSnapshotMonthlyCost, err = s.priceFetcher.GetSnapshotMonthlyCostPerGB(ctx, sourceRegion)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get source snapshot monthly cost: %v", err)
+		}
+	}
+
+	// Fetch the snapshot monthly cost.
+	destSnapshotMonthlyCost, err := s.priceFetcher.GetSnapshotMonthlyCostPerGB(ctx, destRegion)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get dest snapshot monthly cost: %v", err)
+	}
+
+	// Fetch the scanner job monthly cost.
+	scannerPerHourCost, err := s.priceFetcher.GetInstancePerHourCost(ctx, destRegion, scannerInstanceType, marketOption)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get scanner per hour cost: %v", err)
+	}
+
+	// Fetch the scanner job volume monthly cost.
+	scannerRootVolumeMonthlyCost, err := s.priceFetcher.GetVolumeMonthlyCostPerGB(ctx, destRegion, scannerVolumeType)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get volume monthly cost per GB: %v", err)
+	}
+
+	// Fetch the data transfer cost per GB (if source and dest regions are the same, this will be 0).
+	dataTransferCostPerGB, err := s.priceFetcher.GetDataTransferCostPerGB(sourceRegion, destRegion)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get data transfer cost per GB: %v", err)
+	}
+
+	// Fetch the monthly cost of the volume that was created from the snapshot.
+	volumeFromSnapshotMonthlyCost, err := s.priceFetcher.GetVolumeMonthlyCostPerGB(ctx, destRegion, fromSnapshotVolumeType)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get volume monthly cost per GB: %v", err)
+	}
+
+	// 0 in case of same region
+	dataTransferCost := dataTransferCostPerGB * scanSizeGB
+	// 0 in case of same region
+	sourceSnapshotCost := sourceSnapshotMonthlyCost * ((float64(jobCreationTimeSec + scanDurationSec)) / SecondsInAMonth) * scanSizeGB
+
+	destSnapshotCost := destSnapshotMonthlyCost * ((float64(jobCreationTimeSec + scanDurationSec)) / SecondsInAMonth) * scanSizeGB
+	volumeFromSnapshotCost := volumeFromSnapshotMonthlyCost * ((float64(jobCreationTimeSec + scanDurationSec)) / SecondsInAMonth) * scanSizeGB
+	scannerCost := scannerPerHourCost * ((float64(jobCreationTimeSec + scanDurationSec)) / SecondsInAnHour)
+	scannerRootVolumeCost := scannerRootVolumeMonthlyCost * ((float64(jobCreationTimeSec + scanDurationSec)) / SecondsInAMonth) * float64(scannerRootVolumeSizeGB)
+
+	jobTotalCost := sourceSnapshotCost + volumeFromSnapshotCost + scannerCost + scannerRootVolumeCost + dataTransferCost + destSnapshotCost
+
+	// Create the Estimation object base on the calculated data.
+	costBreakdown := []models.CostBreakdownComponent{
+		{
+			Cost:      float32(destSnapshotCost),
+			Operation: string(DestinationSnapshot),
```

Comment:
Since we always have 2 regions that we are moving data between, I think source and dest are understandable enough. 
But I don't mind changing to what you suggested if you think it is more clear

Reply:
I think its more understandable from the users perspective, they don't know/care about the flow inside the provider and how we move snapshots around, they only care about the resources that were created, and we created two snapshots one in region X and one in region Y.